eng/tsp-core/pipelines/pr-tools.yml

```diff
@@ -96,13 +96,13 @@ extends:
                   GH_TOKEN: $(azuresdk-github-pat)
                   VSCODE_DOWNLOAD_URL: $(vscodeUrl)
 
-          - job: change_comment
-            displayName: Describe changes on PR
-            condition: and(succeeded(), eq(startsWith(variables['System.PullRequest.SourceBranch'], 'publish/'), false))
-            steps:
-              - checkout: self
+          # - job: change_comment
```

Comment:
why not just remove?

Reply:
just in case this doesn't doesn't turn out to be better. So can easily revert. One of the reason also to use that in the other repo is to not depend on ADO.

I'll cleanup if this works out well.